### PR TITLE
It has been renamed to `master.componentName`

### PR DESCRIPTION
**Pull request by jhonmike**
```sh
$ helm install jenkins stable/jenkins -f values.yaml
Error: template: jenkins/templates/deprecation.yaml:5:14: executing "jenkins/templates/deprecation.yaml" at <fail "`Master.Name` does no longer exist. It has been renamed to `master.componentName`">: error calling fail: `Master.Name` does no longer exist. It has been renamed to `master.componentName`
```

### Your checklist for this pull request

🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or in [Jenkins JIRA](https://issues.jenkins-ci.org)
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Did you provide a test-case? That demonstrates feature works or fixes the issue.
- [x] Doc

### CODE REVIEW

File: demos/kubernetes-helm/README.md [hunk 1]
```diff
@@ -8,17 +8,17 @@ The latest stable helm chart can be found [here](https://github.com/helm/charts/
...
 ```yaml
-Master:
-  Name: jenkins-master
-  Image: "jenkins/jenkins"
-  ImageTag: "2.134-alpine"
-  ImagePullPolicy: "IfNotPresent"
-  Component: "jenkins-master"
+master:
+  componentName: jenkins-master
+  Image: 'jenkins/jenkins'
```
Comment (timja) [inline demos/kubernetes-helm/README.md]: looking at the [docs](https://github.com/helm/charts/tree/master/stable/jenkins) the rest of fields should be updated as well? they are no longer upper case

Comment (jhonmike) [reply demos/kubernetes-helm/README.md]: truth, I will adjust, thanks for the warning

File: demos/kubernetes-helm/README.md [hunk 2]
```diff
@@ -29,14 +33,14 @@ master:
   # etc.  Best reference is https://<jenkins_url>/configuration-as-code/reference.  The example below creates a welcome message:
   JCasC:
     enabled: true
-    PluginVersion: 1.5
+    pluginVersion: 1.5
```
Comment (timja) [inline demos/kubernetes-helm/README.md]:
```suggestion
    pluginVersion: 1.35
```

File: demos/kubernetes-helm/README.md [hunk 3]
```diff
@@ -29,14 +33,14 @@ master:
...
     SupportPluginVersion: 1.5
```
Comment (timja) [inline demos/kubernetes-helm/README.md]:
```suggestion
```

File: demos/kubernetes-helm/values.yaml [hunk 4]
```diff
@@ -3,8 +3,8 @@
 # Declare name/value pairs to be passed into your templates.
 # name: value
 
-Master:
-  Name: jenkins-master
+master:
+  componentName: jenkins-master
   Image: "jenkins/jenkins"
```
Comment (timja) [inline demos/kubernetes-helm/values.yaml]: same updates should be applied here

**Pull request merged**